appveyor.yml

```diff
@@ -57,4 +55,6 @@ build_script:
 test: off
 
 after_build:
-  - ccache -s
+  # Try to get the info about the cache directory
+  - ccache -s -p
+  - sccache -s -p
```

Comment:
I suppose sccache is not installed here

Reply:
Yeah, not sure what magic I had in mind :D 
Planned to remove with the `-p` flag (or should we keep it?)

Reply:
It doesn't hurt to keep it I think